Procfile

```diff
@@ -0,0 +1,2 @@
+web: bundle exec rails server -p $PORT
+release: bundle exec rake db:migrate db:seed
```

Comment:
Because everytime heroku creates a new app I need to manually migrate and seed before it starts working. It was starting to get boring to have to do that all the time.